portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +23,14 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    ArrayList<String> messages = new ArrayList<>();
+    messages.add("Hello World!");
+    messages.add("This is a message");
+    messages.add("This is another message");
```

Comment:
Did you try running this?  I don't think this will work...

Reply:
It did not. I was thinking it was b/c java.utlil was not imported but I tried importing and got the same issue.